src/main/flight/failsafe.h

```diff
@@ -77,9 +81,9 @@ typedef struct failsafeState_s {
     int16_t events;
     bool monitoring;
     bool active;
-    uint32_t rxDataFailurePeriod;
-    uint32_t validRxDataReceivedAt;
-    uint32_t validRxDataFailedAt;
+    int32_t rxDataFailurePeriod;
+    int32_t validRxDataReceivedAt;
+    int32_t validRxDataFailedAt;
```

Comment:
Why convert these to signed? The `micros()` counter will overflow an `int32_t` in about 35 minutes and go negative.

Reply:
Forget this. 😓 I wanted to keep this delay separated from normal rx recovery but I now see inav just adds both numbers together which is the simplest solution and doesn't yield negative numbers.